src/main/java/bhs/devilbotz/RobotContainer.java

```diff
@@ -93,11 +94,11 @@ public RobotContainer() {
   private void configureBindings() {
 
     new JoystickButton(leftJoystick, 1)
```

Comment:
I know other bindings dont have it, but we should add comments to each binding in order to make it easier to understand the code

Reply:
I agree, we should add comments as part of a separate PR.